metricflow/model/validations/unique_valid_name.py

```diff
@@ -24,10 +27,29 @@
     ValidationIssueType,
     validate_safely,
 )
+from metricflow.object_utils import assert_values_exhausted
 from metricflow.specs import ElementReference
 from metricflow.time.time_granularity import TimeGranularity
 
 
+@enum.unique
+class MetricFlowReservedKeywords(enum.Enum):
+    """Enumeration of reserved keywords with helper for accessing the reason they are reserved"""
+
+    METRIC_TIME = "METRIC_TIME"
```

Comment:
This is also defined in `DataSet.metric_time_dimension_name()`

Reply:
Not anymore!